cf_remote/aramid.py

```diff
@@ -307,7 +312,7 @@ def execute(
     hosts = _hosts_to_host_specs(hosts)
     for i, host in enumerate(hosts):
         port_args = []
-        if host.port != _DEFAULT_SSH_PORT:
+        if host.port != str(_DEFAULT_SSH_PORT):
```

Comment:
This is being done right below.

Reply:
but the equality test wasn't working with the str()